src/scss/04_organisms/form/_fieldset.scss

```diff
@@ -0,0 +1,67 @@
+// Configuration maps
+$fieldsetConfig: (
+  paddingBottom: rem(20px),
+  legend: (
+    borderColor: color(shape),
+    borderWidth: rem(1px),
+    color: color(text, dark),
+    fontSize: small,
+    lineHeight: 100%,
+    marginBottom: rem(10px),
+    paddingBottom: rem(11px),
+    paddingLeft: rem(10px),
+    paddingRight: rem(10px),
+    paddingTop: rem(14px),
```

Comment:
Da mettere in breakpoints

Reply:
Queste sarebbe le regole di default, perchè in breakpoints?